src/query/ast/src/ast/statements/merge_into.rs

```diff
@@ -91,6 +91,7 @@ pub struct MergeIntoStmt {
     pub database: Option<Identifier>,
     pub table_ident: Identifier,
     pub source: MergeSource,
+    pub alias_source: Option<TableAlias>,
```

Comment:
why not call it source_alias and target_alias?

Reply:
well, let me change it.